agents/holo_synth/synth_agent.py

```diff
@@ -0,0 +1,208 @@
+import argparse
+import os
+import time
+
+import numpy as np
+import txaio
+import yaml
+
+ON_RTD = os.environ.get("READTHEDOCS") == "True"
+if not ON_RTD:
+    from holog_daq import synth3
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import Pacemaker, TimeoutLock
+
+
+class SynthAgent:
+    """
+    Agent for connecting to the Synths for holography
```

Comment:
Could use an `Args:` section with the `config_file` argument.

Reply:
Some formatting issues here, the format should be like:
```
"""
Agent for connecting to the Synths for holography.

Args:
    config_file (str): Path to config holography config file
"""
```

(I constantly reference the example here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google)